src/HomeWork/hw7/di/Injector.java

```diff
@@ -0,0 +1,47 @@
+package HomeWork.hw7.di;
+
+import HomeWork.hw7.dao.*;
+import HomeWork.hw7.factory.ClientDaoFactory;
+import HomeWork.hw7.factory.HumanDaoFactory;
+import HomeWork.hw7.handler.ConsoleHandler;
+import HomeWork.hw7.model.Human;
+
+import java.lang.reflect.Field;
+
+public class Injector {
+
+    public static void injectDependency() throws IllegalAccessException {
+        Class consoleHandlerClass = ConsoleHandler.class;
```

Comment:
move this classes out of method to list and make everything generic, just give this method list of classes and it should inject dependencies to each class